### PR TITLE
Revert "#955 stepprocess: feature to call selectStep when isSelected is set to true on one of the SohoStepListItemComponent"

### PR DESCRIPTION
**Pull request by nbcp**
Reverts infor-design/enterprise-ng#956

### CODE REVIEW

**Comment by tmcconechy** [comment]
FYI tests are fixed here https://github.com/infor-design/enterprise-ng/pull/988 needs approval

**Review by pwpatton — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]
Approving pending the tests fix i merged on

**Comment by tmcconechy** [comment]
This is patched in https://www.npmjs.com/package/ids-enterprise-ng/v/9.1.5

**Pull request merged**
